extensions/wikia/EmbeddableDiscussions/styles/avatar.scss

```diff
@@ -1,9 +1,13 @@
 @import 'skins/shared/color';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';
 
-$avatar-size: 42px;
+$avatar-size: 30px;
 
 .embeddable-discussions-module {
 	.avatar-container {
+		background-image: url('/extensions/wikia/GlobalNavigation/images/signin_icon.svg'); /* inline */
+		background-position: center 2px;
```

Comment:
Let's leave a comment why we need to reposition the background (shape of default avatar)